rpc/src/v1/impls/devel.rs

```diff
@@ -79,7 +79,7 @@ where
     }
 
     fn get_state_trie_value(&self, key: H256) -> Result<Vec<Bytes>> {
-        match self.db.get(COL_STATE, &key).map_err(|e| errors::kvdb(&e))? {
+        match self.db.get(COL_STATE, &key).map_err(errors::transaction_core)? {
```

Comment:
Is this correct?

Reply:
Yes, because there is no more kvdb error.
The error becomes io::Error.

Reply:
I think it'll be interpreted as `unknown_error` in `transaction_core`, which has a message `"Unknown error when sending transaction"`.
I think using `errors::core` is better here.

Reply:
Looks good. I changed.